plugins/node/opentelemetry-plugin-dns/src/utils.ts

```diff
@@ -42,7 +42,7 @@ export const setError = (
   }
 
   span.setAttributes(attributes);
-  const status = parseErrorCode(code);
+  const status: Status = { code: StatusCode.ERROR };
```

Comment:
You can add the `message` from next line into the object literal.
Looks better in my opinion.